zenodo/modules/deposit/static/json/zenodo_deposit/deposit_form.json

```diff
@@ -645,6 +653,14 @@
                 "value": "hasPart",
                 "name": "is part of this upload"
               },
+              {
+                "value": "isMetadataFor",
```

Comment:
I'm not sure we can add this, as we need to also support the relatedMetadataScheme attribute then from the [DataCite schema](https://schema.datacite.org/meta/kernel-4.1/)

Reply:
We can possibly split this PR into adding the missing contributor types only since some of the related identifier fields might have special meaning/requirements (e.g. we don't have a `isVersionOf` option since we manage this on our side entirely)